web/locale/en_US/messages.po

```diff
@@ -2624,3 +2624,6 @@ msgstr "Enabling this will require you or another admin with Ticketing Admin per
 
 msgid "Place name must be less than 40 characters."
 msgstr "Place name must be less than 40 characters."
+
+msgid "If you find any issues on this site, want to suggest features to the platform, or even better contribute, you can find all the relevant information over"
+msgstr If you find any issues on this site, want to suggest features to the platform, or even better contribute, you can find all the relevant information over"
```

Comment:
Missing a beginning quote here. Can we change the copy to "Zen is open source, you can contribute code and suggest features to help make it better" - where "contribute code" links to the community-platform repo and "suggest features" links to the issues section with the backlog label. :smiley_cat: